addons/account/models/account_move.py

```diff
@@ -2085,6 +2085,19 @@ def _autopost_draft_entries(self):
         ])
         records.post()
 
+    # FIXME: Clarify me and change me in master
```

Comment:
remove comment and add docstring: aim is just to offer the possibility to duplicate thanks to a button instead of a hidden menu, which is more visible